documentation/src/main/asciidoc/topics/sql/person.sql

```diff
@@ -1,6 +1,6 @@
 CREATE TABLE Person (
   name VARCHAR(255) NOT NULL,
-  picture VARBINARY(255),
+  picture BYTEA,
```

Comment:
This isn't that simple. Unfortunately, it depends upon the DB that is being used. VARBINARY(255) is more popular than BYTEA as that is only Postgres. I am not sure which DB you are trying to tailor the docs to.

https://github.com/infinispan/infinispan/blob/main/persistence/sql/src/test/java/org/infinispan/persistence/sql/AbstractSQLStoreFunctionalTest.java#L393-L401